src/tools/crawler/main.rs

```diff
@@ -25,7 +27,12 @@ fn start_logger(default_level: LevelFilter) {
 
 #[tokio::main]
 async fn main() {
-    start_logger(LevelFilter::DEBUG);
+    start_logger(LevelFilter::INFO);
     let args = Args::parse();
-    debug!("Crawler starting with args: {:?}", args);
+    info!("Crawler starting with args: {:?}", args);
+    let mut crawler = Crawler::new().await;
```

Comment:
could we add some newlines, it's so tight without any

Reply:
Done